sdk/examples/xo_python/sawtooth_xo/xo_client.py

```diff
@@ -140,13 +163,37 @@ def _send_xo_txn(self, name, action, space=""):
         )
 
         batch_list = self._create_batch_list([transaction])
-
-        result = self._send_request(
-            "batches", batch_list.SerializeToString(),
-            'application/octet-stream'
-        )
-
-        return result
+        batch_ids = [b.header_signature for b in batch_list.batches]
+
+        if wait and wait > 0:
+            wait_time = 0
+            start_time = time.time()
+            self._send_request(
+                "batches", batch_list.SerializeToString(),
+                'application/octet-stream'
+            )
+            while wait_time < wait:
+                statuses = self._get_statuses(batch_ids)
+                wait_time = time.time() - start_time
+
+                if all(s == 'COMMITTED' for s in statuses.values()):
+                    return 'Batches committed in {:.6} sec'.format(wait_time)
```

Comment:
Do we want to expose the concept of _"batches"_ to the casual `xo` user? Should this say something like _"Game created in . . ."_? Is reporting back how long at all it took actually important here?